guides/micronaut-oracle-autonomous-db/micronaut-oracle-autonomous-db.adoc

```diff
@@ -143,25 +143,29 @@ resource:application-oraclecloud.yml[]
 
 Create a test to verify that database access works:
 
-test:ThingRepositoryTest[]
+test:repository/ThingRepositoryTest[]
 
 == Testing the Application
 
 There are two options for running the tests; one is to run against the live database, and the other is to run tests locally with an Oracle database in a Docker container using https://www.testcontainers.org/[Testcontainers].
 
-Add the following dependencies:
+Replace the following dependencies:
+
+:dependencies:
+dependency:junit-jupiter[groupId=org.testcontainers,scope=testImplementation]
+:dependencies:
+
+with:
 
 :dependencies:
```

Comment:
and since there's only 1 dependency here you can remove the two `:dependencies:`

Reply:
Thanks